osm2gtfs/core/configuration.py

```diff
@@ -76,7 +76,7 @@ def get_schedule_source(self, refresh=False):
             else:
                 # Check if it is a valid url
                 try:
-                    schedule_source_file = urlopen(source_file)
+                    schedule_source_file = urlopen(source_file).read()
```

Comment:
I think the `read()` method should be called in [line 84](#diff-07e4c70d96a2beb51112609727f78897R84) to keep it kind of consistent with [line 74](#diff-07e4c70d96a2beb51112609727f78897R74).

Reply:
yes, you are right. I'll changed it.